### PR TITLE
PAGE WITH CREATE GOALS BUTTON COMPLETE

### PR DESCRIPTION
**Pull request by Debbyyy**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import AppHeader from './components/header/Header';
+import Create from './components/create-goal';
+
+function App() {
+  return (
+    <div>
+      <AppHeader />
+      <Create></Create>
```
Comment (abiona01) [inline client/src/App.js:8]: This should be self-closing, like the one in line 7

File: client/src/components/create-goal.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function Create() {
+
+    return (
+        <div>
+            {/* typography is the box that contains the image and button
+            button is the green button
+            img tag houses the picture or avatar, whichever one it is */}
+            <Container>
+                <Typography
```
Comment (abiona01) [inline client/src/components/create-goal.js:15]: Typography is for header tags, P and span tags. Import Box form material UI instead.

File: client/src/components/create-goal.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                    style={{ border: '1px solid #c1c1c1', backgroundColor: '#fff', color: '#888', height: '480px', width: '730px', position: 'relative', marginTop: '70px', borderRadius: '7px', textAlign: 'center' }}>
+                    <img src={Avatar}
+                        style={{ height: '37vh', position: 'absolute', top: '50px', right: '260px' }} />
+
+                    
+                    <span style={{ display: 'inline-block', marginTop: '310px' }}>This space is empty. Create a goal to get started.</span>
+                </Typography>
+
+                <Button
+                    style={{ backgroundColor: '#00B87C', position: 'absolute', color: '#fff', marginTop: '-125px', marginLeft: '277px', padding: '12px 45px', borderRadius: '7px', textTransform: 'capitalize' }}>Create goal</Button>
```
Comment (abiona01) [inline client/src/components/create-goal.js:26]: This is alot of inline styling

**Review by abiona01 — CHANGES_REQUESTED** [review]

**Comment by Debbyyy** [comment]
Fixes issue #414

**Comment by KelvinAkpobome** [comment]
You seem to have a conflict, kindly resolve

**Comment by Debbyyy** [comment]
Resolved the conflicts

**Pull request closed without merging**